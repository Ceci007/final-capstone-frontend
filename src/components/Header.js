import React from 'react';
import { Link } from 'react-router-dom';

const Header = () => (
  <header>
    <Link to="/login"><p>Log In</p></Link>
    <Link to="/signup"><p>Sign Up</p></Link>
  </header>
);
export default Header;
