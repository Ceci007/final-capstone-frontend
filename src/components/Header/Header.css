@import "https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap";

header {
  flex-shrink: 0;
  display: flex;
  background: #62b5e5;
  font-family: 'Poppins', sans-serif;
  color: white;
  align-items: center;
  justify-content: center;
}

.title {
  text-align: center;
  letter-spacing: 0.6px;
  font-size: 6vw;
}

.title h4 {
  margin: 10px;
}