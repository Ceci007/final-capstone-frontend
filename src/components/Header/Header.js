import React from 'react';
import './Header.css';

const Header = () => (
  <header>
    <div className="title">
      <h4>Track.it</h4>
    </div>
  </header>
);

export default (Header);
