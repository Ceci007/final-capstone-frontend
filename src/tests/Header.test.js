import { create } from 'react-test-renderer';
import Header from '../components/Header/Header';

test('Header renders correctly', () => {
  const tree = create(
    <Header />,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
