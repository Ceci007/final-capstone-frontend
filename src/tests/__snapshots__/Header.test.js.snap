// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<header>
  <div
    className="title"
  >
    <h4>
      Track.it
    </h4>
  </div>
</header>
`;

exports[`Header renders correctly 1`] = `
<header>
  <div
    className="title"
  >
    <h4>
      Track.it
    </h4>
  </div>
</header>
`;
